docs/user-guide/loadfile.rst

```diff
@@ -0,0 +1,88 @@
+.. _loadfile:
+
+**************************************
+Loading Packet Definitions from a File
```

Comment:
"from a File" => "from a CSV File" (there are some other formats floating around, like XTCE, better to be explicit to avoid confusion)